db/schema.rb

```diff
@@ -0,0 +1,64 @@
+# rubocop:disable all
+# frozen_string_literal: true
+
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_509_191_121) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.string 'text'
```

Comment:
- [ ] I would also like to strongly recommend you change, the Text column datatype into the text instead of a string.  Because the length of a user bio can be long and text is appropriate for that because is variable with unlimited length. Please fix that to make your project works as expected 👍

Reply:
Thanks, @codecaiine. I have implemented the update.